src/components/LocationPageSectionFooter/LocationPageSectionFooter.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Wrapper } from './LocationPageSectionFooter.style';
+
+const LocationPageSectionFooter: React.FC = ({ children }) => {
```

Comment:
i have bigger plans for this shared footer, but punting to a later pr, so for now its just a single wrapping div 🙃